### PR TITLE
Add explicit profile field to General events (cc215)

### PR DESCRIPTION
**Pull request by ViktorHaag**
Explicitly added to profile field to event fixtures as examples.

See also:
- [caliper central issue 215](https://github.com/IMSGlobal/caliper-central/issues/215)

### CODE REVIEW

**Comment by ViktorHaag** [comment]
> Excellent. Perhaps we should add the `profile` property to all v1p2 event fixtures pour encourager les autres.

I will do so and push an update tomorrow.

**Comment by ViktorHaag** [comment]
@arwhyte profile field added to balance of events with commit https://github.com/IMSGlobal/caliper-common-fixtures/pull/230/commits/b982e65ff500236e8d239d6dbc7be4a9bae06eb0

File: v1p2/caliperEventViewViewed.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "@context": "http://purl.imsglobal.org/ctx/caliper/v1p2",
   "id": "urn:uuid:cd088ca7-c044-405c-bb41-0b2a8506f907",
   "type": "ViewEvent",
+  "profile": "GeneralProfile",
```
Comment (arwhyte) [inline v1p2/caliperEventViewViewed.json]: Change to "ReadingProfile:

File: v1p2/caliperEventViewViewedExtended.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "@context": "http://purl.imsglobal.org/ctx/caliper/v1p2",
   "id": "urn:uuid:3a9bd869-addc-48b1-80f6-a14b2ff591ed",
   "type": "ViewEvent",
+  "profile": "GeneralProfile",
```
Comment (arwhyte) [inline v1p2/caliperEventViewViewedExtended.json]: Change to "ReadingProfile"

File: v1p2/caliperEventViewViewedFedSession.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
   "@context": "http://purl.imsglobal.org/ctx/caliper/v1p2",
   "id": "urn:uuid:4be6d29d-5728-44cd-8a8f-3d3f07e46b61",
   "type": "ViewEvent",
+  "profile": "GeneralProfile",
```
Comment (arwhyte) [inline v1p2/caliperEventViewViewedFedSession.json]: Change to "ReadingProfile"

File: v1p2/caliperEnvelopeEventBatch.json [hunk 4]
```diff
@@ -106,6 +108,7 @@
       "@context": "http://purl.imsglobal.org/ctx/caliper/v1p2",
       "type": "ViewEvent",
       "id": "urn:uuid:94bad4bd-a7b1-4c3e-ade4-2253efe65172",
+      "profile": "GeneralProfile",
```
Comment (arwhyte) [inline v1p2/caliperEnvelopeEventBatch.json]: Change to "ReadingProfile"

**Comment by ViktorHaag** [comment]
> See targeted recommendations.

Resolved with commit https://github.com/IMSGlobal/caliper-common-fixtures/pull/230/commits/7e374ec486a60c3229eb4a45e340df8326c1de20

**Pull request merged**